### ISSUE TITLE
Denmark data investigation

### ISSUE THREAD

**Issue by giuliagiupponi** [nomadj1s/bunchingLaborAdjustment#3]
- Read literature to see if there are instances of known bunching by wage earners
- Investigate appropriateness of data sources
- References: Chetty et al (QJE, 2011) and Kreiner et al (AEJ, 2016 link https://pubs.aeaweb.org/doi/pdfplus/10.1257/pol.20140233)

**Comment by giuliagiupponi** [comment]
@nomadj1s 

Having read Kreiner et al (AEJ: EP, 2016) and Chetty et al. (QJE, 2011), here is what I found out about the Danish data and institutional setting:

**Data**: Administrative data  containing third-party information about monthly wages and salaries for all employees at monthly frequency are available starting from Jan 2008. Prior to that date the data is at annual frequency. The monthly administrative data are used by Kreiner et al (2016). Given the administrative nature of the data, I would be quite confident that age in months is available.

**Bunching**: Chetty et al (2011) document bunching at kinks of the income tax schedule among wage earners. However, there is excess mass in the wage earnings distribution only around the cutoff for the top income tax bracket in Denmark, at which the net-of-tax wage rate falls by approximately 30. Smaller kinks exhibit much lower excess mass or no excess mass at all. 

There are three potential concerns in relation to our requirements:
1. Bunching seems relatively small
2. Is it concentrated in the top kink of the income tax schedule. Individuals with earnings around that threshold are unlikely to be liquidity constrained. Hence, consumption smoothing is likely to be less of an issue
3. As documented in the paper, much of the bunching seems to be attributable to firms offering wage-hour contracts that cater workers’ preferences in terms of where to optimally locate on the budget set (aggregate bunching). Individual bunching – i.e. bunching is stemming from individual optimization – seems less relevant.

Overall, I would say that the Danish setting potentially offers the right combination of data and bunching among wage earners. The concern is that bunching is likely occurring at a point of the income distribution that is of limited interest for the research question we have in mind, and that it is not the result of individual optimization.
 
***

I report a brief summary of the papers below:

**Tax Reforms and Intertemporal Shifting of Wage Income: Evidence from Danish Monthly Payroll Records
Claus Thustrup Kreiner, Søren  Leth-Petersen, and Peer Ebbesen Skov
American Economic Journal: Economic Policy 2016, 8(3): 233–257**

**In a nutshell**: This paper uses monthly payroll records for all Danish employees to identify widespread intertemporal shifting of labor income in response to a tax reform that significantly reduced the marginal tax rates for  one-fourth of all employees. 
**Data**: The empirical analysis is based on a new administrative register, known in Denmark as the eIncome register, containing third-party information about monthly wages and salaries for all employees. All firms in Denmark have to report wages and salaries for each employee to the tax agency (SKAT) and tax evasion on wage income is very small (Kleven et al. 2011). The new register has been in place since January 2008 when the frequency of the reporting requirement was changed from yearly to monthly. The register contains the personal registration number of the employee and a firm identifier, which enable us to link the data to various background information of the individual and of the firm from other administrative  registers at Statistics Denmark. 
**Identifying variation**: The identifying variation is provided by a large tax reform in Denmark, which reduced the highest marginal tax rate on earnings from 63 percent to 56 percent, thereby significantly changing incentives for the one-fourth of full-time employees with the highest incomes. The reform was passed in parliament at the end of May 2009 and changed the tax scheme from 2010 onward, thereby creating an incentive for  high-wage earners to postpone wage payments from the end of 2009 to the beginning of 2010. This type of income shifting requires the cooperation of the employer, who reports the earnings to the tax authorities, but is possible without coming into conflict with the tax law. The shifting behavior studied here is therefore a classic example of tax avoidance.
**Results**: We observe a clear negative spike in reported earnings of high-income individuals in the last months of 2009 and a positive spike in the beginning of 2010, and at the individual level we detect taxpayers with a significant drop in reported earnings at the end of 2009 followed by a jump in the beginning of 2010. We detect no systematic effects in other months or for a group of  middle-income individuals with only negligible changes in incentives, confirming that the observed pattern is driven by income shifting.
The large income shifting response at the aggregate level is concentrated on a few taxpayers. Among the employees with an incentive to shift income, we find less than 5 percent exploit the opportunity to shift income, but that these individuals shift large amounts. This observed pattern in reported earnings is difficult to reconcile with intertemporal changes in the timing of work, indicating the observed movements in income are due to tax avoidance rather than real responses.


**Adjustment Costs, Firm Responses, And Micro Vs. Macro Labor Supply Elasticities: Evidence From Danish Tax Records
Raj Chetty John N. Friedman Tore Olsen Luigi Pistaferri
The Quarterly Journal of Economics, May 2011, Vol. 126, No. 2 (May 2011), pp. 749-804**

**In a nutshell**: By analyzing bunching at kinks using Danish tax records, we show that:
1. Larger kinks generate larger taxable income elasticities
2. Kinks that apply to a larger group of workers generate larger elasticities
3. The distribution of job offers is tailored to match workers' aggregate tax preferences in equilibrium

**Data**: The primary dataset is the tax register from 1994-2001, which contains panel data on wage earnings, self-employment income, pensions, capital income and deductions, spouse ID, and several other characteristics. The tax register contains records for more than 99.9% of individuals between the ages of 15-70 in the population. We merge the tax data with the Danish Integrated Database for Labor Market Research (IDA), which includes data on education, firm ID, occupation, labor market experience, and number of children for every person in Denmark.
**Results**: The elasticities implied by the amount of bunching at large kinks are significantly larger than those implied by the amount of bunching at smaller kinks. There is substantial, visually evident excess mass in the wage earnings distribution around the cutoff for the top income tax bracket in Denmark, at which the net-of-tax wage rate falls by approximately 30%. There is little excess mass at kinks where the net-of-tax wage falls by 10%, and no excess mass at kinks that generate variation in net-of-tax wages smaller than 10%. Similarly, we find no changes in earnings around the small tax reforms that change net-of-tax wages by less than 10%.
We demonstrate that firms and unions cater to the tax incentives of the most common workers. Wage hours offers are tailored to the tax preferences of the majority of workers who have small deductions. We label this supply-side response to tax incentives induced by the aggregation of workers' tax preferences “aggregate bunching”.

**Comment by giuliagiupponi** [comment]
One aspect that could make the Danish data potentially more interesting though is if it is possible to link measures of liquidity constraints and of consumption. I'll look into this.

- [ ] Look into possibility of linking data on consumption and liquidity constraints

**Comment by nomadj1s** [comment]
This is great summary. I agree that there may be concerns that the people who are bunching in wages are not the people for whom there are the greatest concerns about welfare losses.

However, here is an alternative interpretation, which is contingent on how the evidence looks:
1. Suppose we do see the bunching is achieved by uneven labor supply throughout the year
2. And/or suppose we can further show that this is facilitated by those with liquid assets, who can smooth consumption
3. Then we uncover, perhaps, and under-explored mechanism by which bunching is achieved: lumpy labor supply at the sub-annual frequency, which requires assets for smoothing consumption. Put another way: intertemporal substitution is higher for those households who can "afford" to do so. That might have implications for welfare, not of the individuals who bunch, but from an optimal tax perspective. It might suggest that taxes among wealthy households who can retime labor supply are more inefficient than we thought, but on the other hand, it might supply that wage subsidies to households who have little wealth are relatively more efficient than we thought (I haven't thought this through fully).

**Comment by nomadj1s** [comment]
@giuliagiupponi:

Here is what Henrik says:

> Thanks for your email. The quick answer to your question is that you need to team up with a person affiliated with an accredited institution in Denmark. I have access via my affiliation with a research center at the U of Copenhagen (but as a foreign-based person there are some restrictions in how my access works); in general it’s best to have at least one co-author who sits in Denmark. It’s possible that, as part of the project, they’d give you remote access (like I have), but they are generally becoming more conservative about doing this. It may be that you won’t be able to work directly on the data, but will have to rely on your Denmark-based co-author to run the code and export the results. All the work is done remotely on servers at Statistics Denmark (even for authors based in Denmark). I’d be happy to chat briefly on skype/zoom, although things are super constrained these days due to covid and having young children at home all the time. But we could set up a time to chat at some point next week.

Do we have any follow-up questions? One possible question:

1. Are there people that he would recommend that we can try to reach out to for collaboration?

Of course, that depends on if we want to move forward with exploration of Denmark data. And we'd also want to think about total number of authors if we are also co-authoring with people to look at the earnings test. Erik Hurst said that co-authorship with an ADP researcher was not necessarily required, but it helps alot to keep the access to the data going for others if we do. In any event, I think the ADP co-author would be fine with being listed as last author.

**Comment by giuliagiupponi** [comment]
@nomadj1s : I agree a natural follow-up question for Henrik would be to recommend someone to reach out for a collaboration. 

In terms of whether to move forward with the Danish data and/or the ADP data, our three options seem to be:

- use ADP data only: the advantage is that the setting is likely more relevant for welfare considerations, the disadvantage is that we have limited scope for investigating mechanisms (i.e. liquidity channels). In terms of co-authors, this option would likely lead to co-authorship with Gelber, Sacks and one ADP researcher (5 authors in total)
- use Danish data only: probably less relevant for welfare, but likely able to investigate liquidity channel. Co-authorship: Gelber, Sacks and Danish researcher (5 authors in total)
- use both ADP and Danish data: ADP data would potentially highlight relevance of the issue in US, Danish data would allow to study liquidity channel. The caveat here is external validity of the Danish setting wrt US one. Coauthorship: Gelber, Sacks, ADP research and Danish researcher (6 authors in total)

I think we would need more info on the Danish setting before being able to make a decision. Do you think it may make sense to have a chat with Henrik not just about data access but also about the Danish setting? I believe he can give us some advice on how appropriate the Danish setting is and how relatable to the US one. With that info, we can probably reach a more informed decision. 

In terms of co-authors, it seems we are oscillating between 5 and 6 people if everyone is on board. They aren't small teams, but there seems to be little difference among the three options in any case. What do you think?

**Comment by nomadj1s** [comment]
I think the co-author count is right, and there is a chance that Gelber won't be able to participate.

We might want to learn more about the Danish setting, but Henrik seemed to signal limited availability. But he left the door open, so maybe we should take him up on it. I will write him back to try to find a time for us three to meet.

**Comment by nomadj1s** [comment]
@giuliagiupponi: I've added notes from our talk with Henrik. You can edit and add anything that I missed.

**Comment by giuliagiupponi** [comment]
- [ ] Thank you email to Henrik

- [ ] Email Katrine Marie Jakobsen kmtj@econ.ku.dk

**Comment by giuliagiupponi** [comment]
- [ ] Read [Le Maire and Schjerning (JPubE, 2013)](https://www-sciencedirect-com.gate3.library.lse.ac.uk/science/article/pii/S0047272713001618)
